From 08080cc243434a81436500b0c2892981cb4367f0 Mon Sep 17 00:00:00 2001
From: Yazan Shhady <yazan.shhady@solid-run.com>
Date: Thu, 7 Jul 2022 13:38:20 +0300
Subject: [PATCH] mx6cuboxi: update the boot envirnoment - add support for
 HB-CBI

When use HB-CBI: set the is_cbi environment variable to be yes (is_cbi=yes)

Note: the defaut value of is_cbi variable is no (is_cbi=no)
---
 include/configs/mx6cuboxi.h | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/include/configs/mx6cuboxi.h b/include/configs/mx6cuboxi.h
index 1c25857296..5e88773eea 100644
--- a/include/configs/mx6cuboxi.h
+++ b/include/configs/mx6cuboxi.h
@@ -49,6 +49,7 @@
 	"ramdiskaddr=0x13000000\0" \
 	"initrd_high=0xffffffff\0" \
 	"ip_dyn=yes\0" \
+	"is_cbi=no\0" \
 	"console=ttymxc0\0" \
 	"bootm_size=0x10000000\0" \
 	"mmcdev=" __stringify(CONFIG_SYS_MMC_ENV_DEV) "\0" \
@@ -73,10 +74,12 @@
 			"setenv fdtprefix imx6dl; fi; " \
 		"if test ${som_rev} = V15; then " \
 			"setenv fdtsuffix -som-v15; fi; " \
+		"if test ${is_cbi} = yes; then " \
+			"setenv cbisuffix -cbi; fi; " \
 		"if test ${has_emmc} = yes; then " \
 			"setenv emmcsuffix -emmc; fi; " \
 		"if test ${board_name} = HUMMINGBOARD2 ; then " \
-			"setenv fdtfile ${fdtprefix}-hummingboard2${emmcsuffix}${fdtsuffix}.dtb; fi; " \
+			"setenv fdtfile ${fdtprefix}-hummingboard2${cbisuffix}${emmcsuffix}${fdtsuffix}.dtb; fi; " \
 		"if test ${board_name} = HUMMINGBOARD ; then " \
 			"setenv fdtfile ${fdtprefix}-hummingboard${emmcsuffix}${fdtsuffix}.dtb; fi; " \
 		"if test ${board_name} = CUBOXI ; then " \
-- 
2.35.3

